packages/ios-client/Sources/FishjamClient/media/Tracks/LocalAppScreenShareTrack.swift

```diff
@@ -1,27 +1,19 @@
 import WebRTC
 
 /// Utility wrapper around a local `RTCVideoTrack` also managing an instance of `VideoCapturer`
-public class LocalAppScreenShareTrack: VideoTrack, LocalTrack {
+public class LocalAppScreenShareTrack: LocalVideoTrack, LocalTrack {
     internal var capturer: AppScreenShareCapturer
-    internal var videoParameters: VideoParameters
-    internal var videoSource: RTCVideoSource
 
     internal init(
         mediaTrack: RTCVideoTrack, videoSource: RTCVideoSource, endpointId: String, metadata: Metadata = Metadata(),
         videoParameters: VideoParameters, capturer: AppScreenShareCapturer
 
     ) {
         mediaTrack.shouldReceive = false
-        self.videoParameters = videoParameters
         self.capturer = capturer
-        self.videoSource = videoSource
-        super.init(mediaTrack: mediaTrack, endpointId: endpointId, rtcEngineId: nil, metadata: metadata)
-    }
-
-    convenience init(mediaTrack: RTCVideoTrack, oldTrack: LocalAppScreenShareTrack) {
-        self.init(
-            mediaTrack: mediaTrack, videoSource: oldTrack.videoSource, endpointId: oldTrack.endpointId,
-            metadata: oldTrack.metadata, videoParameters: oldTrack.videoParameters, capturer: oldTrack.capturer)
+        super.init(
+            mediaTrack: mediaTrack, videoSource: videoSource, endpointId: endpointId, rtcEngineId: nil,
```

Comment:
I was surprised that linter allowed such syntax. But looks like we could enable `lineBreakBeforeEachArgument` rule in `swift-format-config.json` in another PR 🤔